js/main.js

```diff
@@ -101,7 +101,8 @@ dotnetsheff.viewModels.HomeViewModel = function() {
 					}
 			})
 			.done(function(response) {
-				for(var i=response.results.length - 1;i >= 0 ; --i){
+				var maxPreviousEvents = response.results.length - 4;
```

Comment:
@kevbite  In this line, the calculation determines how many previous events load on the home page.  Currently this is set to 4 here in line 104.  To change this, just change the number.